docs/design/security/unix-tmp.md

```diff
@@ -0,0 +1,45 @@
+
+# Unix temporary files
+
+The Unix support for temporary files is different from the Windows model and developers who
```

Comment:
This should explicitly state that the guidance applies to not just files but anything file-like. The .NET diagnostics tools use FIFO/sockets in temp IIRC.

Reply:
Unix domain sockets have different restrictions, and don't necessarily follow this guidance.